### PR TITLE
DOC Linting

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-mfa/issues/531

### CODE REVIEW

**Review by GuySartorelli — CHANGES_REQUESTED** [review]
Why are you deleting a bunch of docs here? This should be renamed to the `docs/en/index.md` and pulled in from here.

**Comment by emteknetnz** [comment]
Have moved to /docs/en and linked to from readme

File: docs/en/index.md
Comment (ChloeSartorelli) [inline docs/en/index.md:1]: Please add title metadata as per all the other PRs.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: docs/en/index.md
Comment (emteknetnz) [reply docs/en/index.md:1]: Sorry forgot to push

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**